lib/constants.js

```diff
@@ -1,6 +1,21 @@
 'use strict'; // eslint-disable-line strict
 
+// The min value here is to manage further backward compat if we
+// need it
+const iamSecurityTokenSizeMin = 128;
```

Comment:
the security token is not a hash, i put 128 for now, if we want to encode some informations inside it, we can